requirements_test.txt

```diff
@@ -3,6 +3,6 @@ pytest
 matplotlib
 pytest
 pytest-cov
-vtk<9.1.0
+vtk<9.1.0;python_version<"3.10"
```

Comment:
So no VTK for Python 3.10?

Reply:
There isn’t anything official at the moment. There are pyvista wheels compiled for 3.10 and a discussion about it here:
https://github.com/pyvista/pyvista/discussions/2064